ethereum_history_api/circuits/lib/src/rlp/rlp_type_test.nr

```diff
@@ -0,0 +1,89 @@
+use crate::rlp::rlp_type::{RlpList, FragmentWithDataType, decode_list};
+use crate::misc::fragment::Fragment;
+use crate::rlp::decode::{LIST, STRING};
+
+pub fn create_rlp_list() -> RlpList<3, 5> {
+    let fragment1: Fragment<3> = Fragment::new_focused([1, 2, 3]);
```

Comment:
Please only specify types when they can't be inferred. In this case - it can

Reply:
Changed. In case of RlpList it always has to be specified.